### PR TITLE
Sass updates - fixed deprecated apis

### PR DESCRIPTION
**Pull request by jlsnow301**
Atomized from #27

- Sass updated
- Removed deprecated APIs
- Fixed errors in a few files (notably inputs)
- Fixed warning from legacy js entry point

### CODE REVIEW

File: .vscode/settings.json [hunk 1]
```diff
@@ -2,5 +2,8 @@
   "editor.codeActionsOnSave": {
     "source.fixAll.eslint": "explicit"
   },
+  "[yaml][markdown][html][scss]": {
+    "editor.defaultFormatter": "esbenp.prettier-vscode"
```
Comment (ZeWaka) [inline .vscode/settings.json:6]: i assume you'll later replace this with biome in #27

File: package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "tgui-core",
-  "version": "1.2.0",
+  "version": "1.2.1",
```
Comment (ZeWaka) [inline package.json:3]: Updating the version is now done via Release, per https://github.com/tgstation/tgui-core#releasing-a-new-version

**Review by ZeWaka — COMMENTED** [review]
Looks good otherwise, I just did this api change at work lol

File: .vscode/settings.json (hunk 1 above)
Comment (jlsnow301) [reply .vscode/settings.json:6]: yes

Comment (jlsnow301) [reply .vscode/settings.json:6]: biome cannot format these files, so i'm setting these to make it cooperate with prettier when i do add it

**Review by ZeWaka — APPROVED** [review]

**Pull request merged**